Common/log.cpp

```diff
@@ -9,12 +9,12 @@
 
 #include <fstream>
 #include "utils.h"
-#include <boost/iostreams/stream.hpp>
-#include <boost/iostreams/device/null.hpp>
 #include <codecvt>
 #include <fstream>
 
-static std::ofstream _logFile;// = bofstream();
+std::ofstream Log::_logFile;// = bofstream();
+
+std::ostream* Log::_nullStream = &std::cout;
```

Comment:
It's just the default. Desktop and Engine set up the Log class by setting the _nullstream with the boost null_sink.
This is done to avoid Common to include the boost library.

Reply:
okidoki